src/Lean/Elab/MutualDef.lean

```diff
@@ -68,6 +64,8 @@ private def check (prevHeaders : Array DefViewElabHeader) (newHeader : DefViewEl
     throwError "'partial' theorems are not allowed, 'partial' is a code generation directive"
   if newHeader.kind.isTheorem && newHeader.modifiers.isNoncomputable then
     throwError "'theorem' subsumes 'noncomputable', code is not generated for theorems"
+  if newHeader.modifiers.isNoncomputable && newHeader.modifiers.isUnsafe then
+    throwError "'noncomputable unsafe' is not allowed"
```

Comment:
A merge at some point appears to have gone wrong.

Reply:
Oh, looking further, the are other changes not incorporate here either. I presume you have just not merged master in a while. Sorry for the noise!

Reply:
No, you were completely right! Thanks for flagging.